.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/gnosis/safe-contracts"]
+	path = external/gnosis/safe-contracts
+	url = git@github.com:OpenSTFoundation/safe-contracts.git
```

Comment:
does git submodule allow to check out a version, or should this be done out of band with a shell script?

Reply:
It is handled with git submodules. Once copied and synced with the branch the appropriate version of gnosis safe (0.1.0) would be checked out.